notebooks/quickstart.ipynb

```diff
@@ -10,12 +10,12 @@
     "\n",
     "\n",
     "----------------\n",
-    "## Quickstart\n",
-    "### Creating and visualizing a `spaghetti.Network` object\n",
+    "# Quickstart\n",
+    "## Creating and visualizing a `spaghetti.Network` object\n",
     "\n",
     "**Author: James D. Gaboardi** **<jgaboardi@gmail.com>**\n",
     "\n",
-    "**This notebook is a basic walk-through for:**\n",
+    "**This notebook provides an explanation of network creation followwed by an emprical examples for:**\n",
```

Comment:
typo 'followed', singular example

Reply:
> I added some additions to the text. Let me know if you prefer me typing it this way or adding then with an additional commit next time.

Awesome! I can go ahead and integrate your comments into this PR.

> I think overall it's nice! We can work towards integrating the plot functionality in splot if you like? I am just wondering if we integrate it as a .plot() method which spaghetti objects would you suggest to assign it too?

This is an important question and one that I hadn't even though of yet. It would be nice to have it on the `Network` object, but that raises some questions:
1. The `.plot()` method would have to use [`element_as_gdf`](https://pysal.org/spaghetti/generated/spaghetti.element_as_gdf.html#spaghetti.element_as_gdf) to accommodate plotting functionality
2. The point observations associated with the `Network` object are technically their own class ([`PointPattern`](https://pysal.org/spaghetti/generated/spaghetti.PointPattern.html#spaghetti.PointPattern)), but the exist within an attribute of `Network` (e.g. [`Network.pointpatterns`](https://pysal.org/spaghetti/notebooks/pointpattern-attributes.html#1.-Allocating-observations-(snapping-points)-to-a-network:)).

Thoughts? Either way this seems complicated, but maybe I'm wrong?